### DISCUSSION TITLE
Display Pivot points

### DISCUSSION THREAD

**Discussion by botradingblog1** [kernc/backtesting.py#816] [Q&A] [answered]
Hi,
Thanks for creating this awesome backtesting lib. I love it!

I'm looking for a way to display pivot points on the main chart. So I have added an indicator that has np.nan values and close price values for the pivots but it is not displaying on the main chart. Instead, a separate chart is displayed that doesn't show any values at all.

Any suggestions greatly appreciated.
Thanks!

**Comment by kernc** [comment, marked as answer]
I think in your case the separate chart might be displaying small circles, but they are few and tiny?

In any event, you should be able to use [`Strategy.I`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.I) with params:
```py
self.I(indicator_func, ..., scatter=True, overlay=True)
```
This will plot indicator as scatter plot (non-nan values are points) and overlay it over OHLC plot.

**Reply by botradingblog1** [reply]
Awesome, will try that out. Thanks!

**Reply by botradingblog1** [reply]
I didn't see any dots on the second chart but perhaps my eyes aren't that good anymore. :)
